srunner/scenarios/opposite_vehicle_taking_priority.py

```diff
@@ -133,24 +133,44 @@ def _create_behavior(self):
             self.other_actors[0], self.ego_vehicle, location_of_collision_dynamic)
         sync_arrival_stop = InTriggerDistanceToVehicle(self.other_actors[0],
                                                        self.ego_vehicle,
-                                                       15)
+                                                       41)
         sync_arrival_parallel.add_child(sync_arrival)
         sync_arrival_parallel.add_child(sync_arrival_stop)
 
-        keep_velocity_for_distance = py_trees.composites.Parallel(
-            "Keep velocity for distance",
+        # Generate plan for WaypointFollower
+        waypoint = self.other_actors[0].get_world().get_map().get_waypoint(self.other_actors[0].get_location())
```

Comment:
get_map() is now available in CarlaDataProvider.

Reply:
Fixed.